<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>430</y>
      <width>162</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>561</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>360</y>
       <width>681</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Предыдущий персонаж</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>Изменение базы данных</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Следующий персонаж</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>681</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="picture">
        <property name="text">
         <string>Picture</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame">
          <property name="styleSheet">
           <string notr="true">background: #a8d7e1; /* Цвет блока */
  width: 100%; /* Ширина блока */
  height: 350px;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QWidget" name="">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>25</y>
             <width>181</width>
             <height>171</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_53">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Имя</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="name">
                <property name="text">
                 <string>name</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_14">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Место рождения</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="origin_name">
                <property name="text">
                 <string>место рождения</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>ID</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="id">
                <property name="text">
                 <string>ID</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Тип</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="type">
                <property name="text">
                 <string>тип</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QWidget" name="">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>20</y>
                 <width>161</width>
                 <height>171</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_12">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Пол</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="gender">
                    <property name="text">
                     <string>пол</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_15">
                  <item>
                   <widget class="QLabel" name="label_11">
                    <property name="text">
                     <string>Статус</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="status">
                    <property name="text">
                     <string>статус</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_13">
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Место проживания</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="local_name">
                    <property name="text">
                     <string>место проживания</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_8">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Раса</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="race">
                    <property name="text">
                     <string>раса</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
